CDCgov/cdc-open-viz
# Feature/dev 4215h

## [Replace With Ticket Number]



## Testing Steps
Select Chart Type: Select the chart to a Bar, Line, or Combo type.
Select Scale Type: Choose the "Date or Time scale" option for the chart.
Configure Time Scale: Adjust the time scale settings as needed.
Enable Brush Slider:
On the Date Category Panel, locate a new checkbox labeled "Brush Slider".
Check this box to enable the Brush Slider. A brush slider should appear below the charts.
Adjust Brush Slider:
Drag the corners of the brush slider to precisely adjust the scale of the bars or lines in the chart.
Update Legend Position:
Change the legend position to the bottom of the chart and verify that it does not overlap with the brush slider.
Adjust Tick Rotation:
Modify the rotation of the ticks on the axis to ensure there is no overlap with the brush slider.
Activate Filters:
Turn on any filters. The brush slider should reset to its initial state once the filters are activated.




## Self Review

- I have added testing steps for reviewers
- I have commented my code, particularly in hard-to-understand areas
- My changes generate no new warnings
- New and existing unit tests are passing

## Screenshots (if applicable)



## Additional Notes
